charts/backstage/ci/topologySpreadConstraintsString-values.yaml

```diff
@@ -0,0 +1,9 @@
+backstage:
+  topologySpreadConstraints: |-
```

Comment:
Same as above. The type is an array, not a string, therefore I would advise against suggesting that this value is supported.